### ISSUE TITLE
About edgeloss

### ISSUE THREAD

**Issue by chenhaiwen** [Chnja/RDPNet#5]
the function "loss = combine_loss(prediction, target)", this loss include edge loss, focal loss and dice loss or only have edge loss?

**Comment by chenhaiwen** [comment]
The meaning of KSIZE?

**Comment by Chnja** [comment]
combine_loss includes focal loss, dice loss and edge loss
KSIZE can be approximated as the radius of area _N_
